_rules/SC1-1-1-accessible-name-is-image-file-name.md

```diff
@@ -0,0 +1,77 @@
+---
+name: image accessible name contains image file name
+description: |
+  This rule checks that the use of an image's file name in it's accessible name, is appropriate.
```

Comment:
I suggest we put something about authoring tools in the description. Make it clear why this is a thing.